### ISSUE TITLE
Live updated map don't display; OsmAnd shows "index not supported" error

### ISSUE THREAD

**Issue by raindropsfromsky** [osmandapp/Osmand#2783]
Reporting for OsmAnd 2.4.0#7374MQA

I am using OnePlus 2 (OxygenOs 3.0.2, based on Android M 6.0.1)

I have side-loaded the obf files (world overview, India) in the map folder.
I have enabled **Map Live update** option.

In the **Manage map files > Local** tab, the maps are displayed in green, which means the maps are being updated correctly.

Yet the maps do not get displayed on screen.
After a couple of seconds, the screenshows a list of errors in the following format:

The version of 'India_Asia.obf' is not supported.
(This is followed by similar lines, each mentioning a  different map fie name.)

What could be the problem?

**Comment by raindropsfromsky** [comment]
I installed the different version, which gets installed as -

OsmAndF~2.4.0#13478D.
Although this app has a different folder to hold the map, the problem does not vanish.

**Comment by raindropsfromsky** [comment]
I uninstalled both the versions mentioned above, and deleted the data folders.
Then I re-installed the second app (OsmAndF~2.4.0#13478D).
During installation, I selected "External data storage" path.
The data folder path reads as: /storage/emulated/0/Android/data/net.osmand.dev/files.
Finally I downloaded the **World_basemap** and **India_Asia** maps from within OsmAnd.
I confirmed that these obf files are at: Internal Storage/Android/data/net.osmand.dev/files

Now OsmAnd works fine.

The newly downloaded maps are as follows:
**World_basemap** 261 MB
**India_Asia** 566 MB

But the maps that were live-updated are 186 MB and 371 MB, respectively.
In other words, the live-updated files are much smaller.

**Conclusion 1:** The **live map update** feature corrupted the map files.

I enabled the **live update** feature again, and OsmAnd downloaded three India maps (total: 2.7 MB). But the file sizes did not change, and the app works fine with the updated map files. 

**Conclusion 2:** The live update does not corrupt the map files consistently.

**Comment by raindropsfromsky** [comment]
On further thinking, I realized that I had never used the external data storage option before today.
Is this a factor? 
(When the data is stored in the system-protected folder, can live update corrupt it?)

**Comment by vshcherb** [comment]
The default file location storage has changed recenlty. The issue with osm live is not reproducible
